src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/sizes/recordcomponentnumber/InputRecordComponentNumberDifferentMaxForScope.java

```diff
@@ -0,0 +1,140 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.sizes.recordcomponentnumber;
+
+import java.awt.Point;
+import java.awt.Shape;
+import java.util.ArrayDeque;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+import org.w3c.dom.Node;
+
+/* Config:
+ *
+ * public "max" = 5
```

Comment:
lets avoid this.
in future we will load config from this comment, this approach will complicate everything.

single instance of Check should be used for file.

Reply:
Can I just remove this test then?  The point of this one was to prove that we can use multiple configs for different access modifiers.

Reply:
yes, just  remove test

Reply:
Removed and added  a new one, set to only "private" access modifier.  This was required for test coverage.